tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -90,7 +90,7 @@ public SecuritySettings(IConfigurationSource? source, IConfigurationTelemetry te
             ApiSecurityEnabled = config.WithKeys(ConfigurationKeys.AppSec.ApiExperimentalSecurityEnabled)
                                        .AsBool(false);
             UseUnsafeEncoder = config.WithKeys(ConfigurationKeys.AppSec.UseUnsafeEncoder)
-                                     .AsBool(false);
+                                     .AsBool(true);
```

Comment:
i dont think we should activate it just yet but see some reliability environment data from the aas apps